lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -160,20 +161,36 @@ data class SyncingResponse(
     override val highestBlockNumber: Int,
 ) : Syncing()
 
+sealed interface GetBlockWithResponse {
```

Comment:
Also, we can probably move all of these to `lib/src/main/kotlin/com/swmansion/starknet/data/types/Block.kt`

Reply:
#432